src/tools/fuzzing/fuzzing.cpp

```diff
@@ -196,6 +196,7 @@ void TranslateToFuzzReader::build() {
 void TranslateToFuzzReader::setupMemory() {
   // Add memory itself
   MemoryUtils::ensureExists(&wasm);
+  auto& memory = wasm.memories[0];
```

Comment:
Would it make sense to stop special-casing memory 0 entirely and treat memories uniformly?

Reply:
Perhaps, but I'm not sure we want full uniformity. We need some special conditions on the memories we use, like being of a reasonable known size (so we don't constantly have traps). It might actually be good that we just modify one memory atm to that shape and not all of them, but it does mean our fuzzing of the others is limited in other ways.

For imports, though, we just need to modify them all, or else we immediately trap on any use.

Reply:
Even if we want to occasionally be more permissive, there's no reason memory 0 should be special. I'm concerned that treating it specially in the fuzzer will mask bugs where we inappropriately treat it as special elsewhere.

Reply:
Agreed, yeah, it has downsides. Worth considering separately from this PR I think (which does not make us more specific to memory 0).